src/sentry/auth/access.py

```diff
@@ -319,21 +336,184 @@ def has_any_project_scope(self, project: Project, scopes: Collection[str]) -> bo
 
         return False
 
-    def to_django_context(self) -> Mapping[str, bool]:
-        return {s.replace(":", "_"): self.has_scope(s) for s in settings.SENTRY_SCOPES}
 
+@dataclass
+class ApiAccess(Access):
```

Comment:
A semantic afterthought... I see how `ApiAccess` follows the same naming convention as `ApiOrganization` and friends, but it could be unfortunately misleading if read as "access to an API" or "access for an API request". Perhaps it should be named `ApiBackedAccess` for symmetry with the existing `DatabaseBacked*` services, and we could change `DbAccess` to `DatabaseBackedAccess` for good measure.

Reply:
Yea, that's reasonable and easy to change.

Reply:
Changed